pkg/sentry/events.go

```diff
@@ -0,0 +1,86 @@
+package sentry
+
+import (
+	"fmt"
+	"net/url"
+	"strconv"
+	"time"
+)
+
+var reqFields = [...]string{
+	"id",
+	"title",
+	"project",
+	"project.id",
+	"release",
+	"count()",
+	"epm()",
+	"last_seen()",
+	"failure_rate()",
+	"level",
+	"event.type",
+	"platform",
+}
+
+type SentryEvents struct {
+	Data []SentryEvent          `json:"data"`
+	Meta map[string]interface{} `json:"meta"`
+}
+
+type SentryEvent struct {
+	ID              string    `json:"id"`
+	Title           string    `json:"title"`
+	Project         string    `json:"project"`
+	ProjectId       int64     `json:"project.id"`
+	Release         string    `json:"release"`
+	Count           int64     `json:"count()"`
+	EventsPerMinute float64   `json:"epm()"`
+	LastSeen        time.Time `json:"last_seen()"`
+	FailureRate     float64   `json:"failure_rate()"`
+	Level           string    `json:"level"`
+	EventType       string    `json:"event.type"`
+	Platform        string    `json:"platform"`
+}
+
+type GetEventsInput struct {
+	OrganizationSlug string
+	ProjectIds       []string
+	Environments     []string
+	Query            string
+	From             time.Time
+	To               time.Time
+	Sort             string
+	Limit            int64
+}
+
+func (gii *GetEventsInput) ToQuery() string {
```

Comment:
Nit: I think `gei` is supposed to be the acronym of `GeEventsInput`
```suggestion
func (gei *GetEventsInput) ToQuery() string {
```